wsgi.py

```diff
@@ -0,0 +1,16 @@
+"""
+WSGI config for lambomoon project.
+
+It exposes the WSGI callable as a module-level variable named ``application``.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/3.2/howto/deployment/wsgi/
+"""
+
+import os
+
+from django.core.wsgi import get_wsgi_application
+
+os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'lambomoon.settings')
```

Comment:
I think your app breaks because of this: `lambomoon.settings` should only be `settings`, and check for other occurrences too, please.